desktop/appcontainer/FeedbackDialog.js

```diff
@@ -38,7 +38,9 @@ export class FeedbackDialog extends Component {
             items: [
                 textArea({
                     placeholder: 'Please enter your comments...',
-                    style: {height: 250, marginBottom: 2},
+                    width: null,
```

Comment:
I am rushing a bit and reading this on github / not running the code, but I see a  few cases here and below where we are setting the width to null on these components.  Why is that?  It seems like, as a general goal, that should be a pretty unusual thing to need to do.